packages/components/src/vue/vuex-modules/data/auth-module.js

```diff
@@ -133,7 +133,7 @@ AuthModule.api = (moduleInstance, store) => {
       }
       moduleInstance._auth.session().then((data) => {
         moduleInstance._api.updateAuthData(data)
-      }).catch((error) => { // eslint-disable-line handle-callback-err
+      }).catch((error) => { // eslint-disable-line node/handle-callback-err
```

Comment:
@kirlat and @balmas , I believe we have some problems with building here.
Because with this statement `node/handle-callback-err` my build process fails.
I was to remove `node/` to be able to build the package.
Why do you adding `node/` ? Do you hav the opposite problem with the build?

Reply:
The `node` prefix is what eslint requires me to have in order to build the package. I believe this is because either ESLint or its rules packages have been updated.

I'm wondering if you have an outdated version of dependencies. Can you please check that your ESLint/rules packages are matching those listed in the `package.json` of components? I.e.:
```
"eslint": "^7.12.1",
"eslint-config-standard": "^16.0.1",
"eslint-plugin-import": "^2.22.1",
"eslint-plugin-jsdoc": "^30.7.7",
"eslint-plugin-node": "^11.1.0",
"eslint-plugin-promise": "^4.2.1",
"eslint-plugin-standard": "^4.0.2",
"eslint-plugin-vue": "^7.1.0",
"eslint-scope": "^5.1.1"
```
I'm wondering if this issue occurred because of the dependencies mismatch. `npm run bootstrap` from the root of `alpheios-core` should probably fix that.

Reply:
Yes, it is quite the same - I uploaded your branch and didn't change anything.
I don't think that the problem is from libs versions.

Reply:
Hmm... Let me investigate this further.

Reply:
I uploaded your branch, reinstalled the dependencies - and build, for now i don't have problems with building.
I believe that there was some problems with updating dependencies through lerna.
I have no additional questions here!

Reply:
Lerna updates can be tricky. Sometimes it updates everything, but sometimes it misses something out. I still did not figure out why is that happening 🙁, so in a moment of doubt I go to the package locations (`node_modules`) and check the versions in `package.json` manually.